src/dcp1/dcp1.c

```diff
@@ -268,11 +268,11 @@ void DCOPY_print_usage(void)
     printf("  -d, --debug <level> - specify debug verbosity level (default info)\n");
     printf("  -f, --force         - delete destination file if error on open\n");
     printf("  -p, --preserve      - preserve permissions, ownership, timestamps, extended attributes\n");
-    printf("  -p, --verbose       - verbose output\n");
-    printf("  -p, --quiet         - quiet output\n");
+    printf("  -v, --verbose       - verbose output\n");
+    printf("  -q, --quiet         - quiet output\n");
     printf("  -s, --synchronous   - use synchronous read/write calls (O_DIRECT)\n");
-    printf("  -k, --chunksize     - specify chunksize in MB unit (default 1MB)\n");
-    printf("  -b, --blocksize     - specify blocksize in MB unit (default 1MB)\n");
+    printf("  -k, --chunksize     - work size per task in MB (default 1MB)\n");
+    printf("  -b, --blocksize     - IO buffer size in MB (default 1MB)\n");
```

Comment:
The other confusing aspect is that the `dcp/dcp1.c` usage message also says that `--chunksize` and `--blocksize` are specified in MB (as above), when they are actually in units of bytes.  I've corrected the usage message there to match the actual behaviour.

Reply:
Thanks!